internal/receiver/awscontainerinsightreceiver/host/ebsvolume.go

```diff
@@ -43,7 +43,7 @@ func NewEbsVolume(instanceId string, refreshInterval time.Duration, logger *zap.
 
 	//add some sleep jitter to prevent a large number of receivers calling the ec2 api at the same time
 	time.Sleep(hostJitter(3 * time.Second))
-	ebsVolume.refresh()
+	ebsVolume.refresh(region)
```

Comment:
Could you make `region` as a field for ebsVolume so that you don't need to pass the parameter in `refresh()` function?